examples/simple_example/main.tf

```diff
@@ -117,4 +117,15 @@ module "healthcare" {
       }]
     }
   ]
+  consent_stores = [
```

Comment:
Also add checks in https://github.com/terraform-google-modules/terraform-google-healthcare/blob/master/test/integration/simple_example/controls/gcloud.rb similar to other ones to check the consent store is successfully created.